graphenecommon/amount.py

```diff
@@ -177,7 +177,8 @@ def __float__(self):
         return float(self["amount"])
 
     def __int__(self):
-        return int(self["amount"] * 10 ** self["asset"]["precision"])
+        # this needs rounding, e.g. 5.1 * 10 ** 5 == 509999.99999999994
+        return int(round(self["amount"] * 10 ** self["asset"]["precision"]))
```

Comment:
Sorry for being a nitpicker, but please use `int()` intead of `round()`.

Reply:
```
int(509999.99999999994) = 509999
round(509999.99999999994) = 510000
```